wire/.editorconfig

```diff
@@ -0,0 +1,31 @@
+; This file is for unifying the coding style for different editors and IDEs.
+; More information at http://editorconfig.org
+
+
+root = true
+
+
+[*]
+charset = utf-8
+end_of_line = lf
+
+
+[*.{php,inc,module}]
+tab_width = 4
```

Comment:
Yep, you are right. We can let tab_width out.
I think I got tricked by my editor, what shows me *.css files with tabs only 2 spaces long in PW wire directories (**without** editorconfig files!). Where my personal indentation is for all files the same (tab = 4 spaces) and thats how I have setup my editor. I think the editor derives that from top-comments spaces or something other (weird) interpretion. :-/